system/settings/settings.xml

```diff
@@ -1712,9 +1712,9 @@
         </setting>
         <setting id="services.webserverport" type="integer" parent="services.webserver" label="730" help="36329">
           <level>1</level>
-          <default>8080</default>
+          <default>0</default>
           <constraints>
-            <minimum>1</minimum>
+            <minimum>0</minimum>
```

Comment:
what about using public port range?

Reply:
In this case 0 lets the library know to to use a random port via the OS.